### PR TITLE
LCOW: (Experimental) Require RS5+ builds

### PR DESCRIPTION
**Pull request by lowenna**
Signed-off-by: John Howard <jhoward@microsoft.com>

This moves the requirement for the experimental LCOW feature to at least RS5 builds of Windows, as per https://github.com/moby/moby/issues/33850#issuecomment-478192332.

The current LCOW solution uses the HCS "v1" schema. With the move to a containerd runtime, we are no longer planning on supporting LCOW support in the v1 schema in opengcs, and have been investing heavily in LCOW support through the v2 schema (driven via containerd, through hcsshim, and onto HCS; called from CRI, Kube, Docker,....).

Phase 2 of this will be to move the graphdriver remote filesystem access away from v1 as well.
Phase 3 will be the deprecation of the "local" libcontainerd runtime in favour of containerd which will only use the "v2" HCS schema.

@dmcgowan @crosbymichael @thaJeztah @jterry75 FYI.

### CODE REVIEW

File: pkg/system/init_windows.go [hunk 1]
```diff
@@ -15,10 +15,10 @@ var (
 	containerdRuntimeSupported = false
 )
 
-// InitLCOW sets whether LCOW is supported or not
+// InitLCOW sets whether LCOW is supported or not. Requires RS5+
 func InitLCOW(experimental bool) {
 	v := GetOSVersion()
-	if experimental && v.Build >= 16299 {
+	if experimental && v.Build >= 17763 {
```
Comment (thaJeztah) [inline pkg/system/init_windows.go]: there's cool constants for this now 😅 https://github.com/moby/moby/pull/39100

Comment (jhowardmsft) [reply pkg/system/init_windows.go]: I didn't realise that was merged yet. Updated.

**Review by thaJeztah — APPROVED** [review]
SGTM

**Comment by jterry75** [comment]
LGTM! (not a maintainer)

File: pkg/system/init_windows.go [hunk 2]
```diff
@@ -3,6 +3,7 @@ package system // import "github.com/docker/docker/pkg/system"
 import (
 	"os"
 
+	"github.com/Microsoft/hcsshim/osversion"
```
Comment (thaJeztah) [inline pkg/system/init_windows.go]: oh! needs a re-run of `vndr` as they're not yet in `vendor`

```
16:44:59 INFO: Building daemon...
16:45:05 ..\..\pkg\system\init_windows.go:6:2: cannot find package "github.com/Microsoft/hcsshim/osversion" in any of:
16:45:05 	c:\go\src\github.com\docker\docker\vendor\github.com\Microsoft\hcsshim\osversion (vendor tree)
16:45:05 	C:\go\src\github.com\docker\docker\vendor\github.com\Microsoft\hcsshim\osversion
16:45:05 	c:\go\src\github.com\Microsoft\hcsshim\osversion (from $GOROOT)
16:45:05 	C:\go\src\github.com\Microsoft\hcsshim\osversion (from $GOPATH)
```

Comment (thaJeztah) [reply pkg/system/init_windows.go]: Or a merge of my PR 😇

Comment (jhowardmsft) [reply pkg/system/init_windows.go]: Fixed

**Review by thaJeztah — APPROVED** [review]
LGTM

**Review by dmcgowan — APPROVED** [review]
LGTM

**Review by tonistiigi — APPROVED** [review]

**Pull request merged**